packages/react/src/Overlay/Overlay.dev.stories.tsx

```diff
@@ -89,3 +89,145 @@ export const SxProps = (args: Args) => {
     </Box>
   )
 }
+
+export const PreventFocusOnOpen = (args: Args) => {
```

Comment:
Most of this story is copy/pasted from existing Overlay stories. Key changes:
* removed focus trap (present in most other stories, something we might want to consider removing)
* inclusion of undocumented `preventFocusOnOpen` prop